mobile2/src/components/BookList.tsx

```diff
@@ -6,6 +6,7 @@ import { BookList_books } from "./__generated__/BookList_books.graphql";
 
 type Props = {
   books: BookList_books;
+  status: String;
```

Comment:
TS string types aren't capitalized

```suggestion
  status: string;
```